__tests__/account.test.ts

```diff
@@ -733,16 +748,16 @@ describe('deploy and test Wallet', () => {
         casm: compiledHelloSierraCasm,
       });
 
-      const innerInvokeEstFeeSpy = jest.spyOn(account.signer, 'signTransaction');
+      // const innerInvokeEstFeeSpy = jest.spyOn(account.signer, 'signTransaction');
       const result = await account.estimateInvokeFee({
         contractAddress: ddc1.deploy.address,
         entrypoint: 'increase_balance',
         calldata: [100],
       });
 
       expect(result).toMatchSchemaRef('EstimateFee');
-      expect(innerInvokeEstFeeSpy.mock.calls[0][1].version).toBe(feeTransactionVersion);
-      innerInvokeEstFeeSpy.mockClear();
+      // expect(innerInvokeEstFeeSpy.mock.calls[0][1].version).toBe(feeTransactionVersion);
```

Comment:
maybe good to add some TODO here?

Reply:
ed2c5ee5f6acf2f2ae8429878a5c59962027a93e